extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -94,9 +94,15 @@ describe('amp-a4a', () => {
         sandbox.spy(AmpA4A.prototype, 'onAmpCreativeRender');
     getSigningServiceNamesMock.returns(['google']);
     xhrMockJson.withArgs(
-        'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
-        {mode: 'cors', method: 'GET'})
-    .returns(Promise.resolve({keys: [JSON.parse(validCSSAmp.publicKey)]}));
+      'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
+      {
+        mode: 'cors',
+        method: 'GET',
+        requireAmpResponseSourceOrigin: false,
+        disableAmpSourceOrigin: true,
```

Comment:
I still think these 2 boolean names are confusing.
We should have the new boolean as `ampCors`, which defaults to `true`. In case it's `false`, it will override `requireAmpResponseSourceOrigin` to false.

As I mentioned, the `requireAmpResponseSourceOrigin` boolean exists only for backward compatibility. No new code should use it as `false`. They should instead go with `ampCors: false`.

@cramforce